tests/structures/test_list_comprehension.py

```diff
@@ -12,8 +12,8 @@ def test_syntax(self):
 
     def test_list_comprehension_with_if_condition(self):
         self.assertCodeExecution("""
-            x = [1, 2, 3, 4, 5]
-            print([v**2 for v in x if v % 2 == 0])
+            print([v for v in range(100) if v % 2 == 0])
+            print([v for v in range(100) if v % 2 == 0 if v % 3 == 0])
```

Comment:
Could you add some tests here for more than 2 conditions, just to increase confidence in the test suite? (e.g. adding `if v > 10`, `if v < 80`)

Reply:
Done!